infra/src/main/kotlin/fi/fta/geoviite/infra/configuration/CacheConfiguration.kt

```diff
@@ -13,12 +12,14 @@ import org.springframework.cache.caffeine.CaffeineCacheManager
 import org.springframework.cache.support.NoOpCacheManager
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
+import java.time.Duration
 
 const val CACHE_GEOMETRY_PLAN = "geometry-plan"
 const val CACHE_GEOMETRY_SWITCH = "geometry-switch"
 
 const val CACHE_ROLES = "roles"
-const val CACHE_COORDINATE_SYSTEMS = "coordinate-systems"
+const val CACHE_GVT_COORDINATE_SYSTEMS = "gvt-coordinate-systems"
```

Comment:
Koordinaattijärjestelmien tietoja haetaan sekä yksittäin srid:llä että kaikki-nipussa geoviitteen "käyttämille" järjestelmille. Tuo on erillinen siksi että yksittäisen haulla löytyy siis paljon järjestelmiä joita ei geoviitteessä normaalisti käytetä (ei halutaa drop-down valintoihin) mutta jotka ovat kyllä valideja (voidaan muuntaa tai näyttää niiden tiedot jos esim jossain suunnitelmassa selllainen on).

Yksittäisen haulla ei tähän asti ollut cachea, mutta ei ole mitään syytä miksi sitä ei voisi kakuttaa ikuisesti.